src/EditorFeatures/CSharpTest/Diagnostics/AddExplicitCast/AddExplicitCastTests_FixAllTests.cs

```diff
@@ -0,0 +1,1743 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
do you have tests with nested cases? i.e.:

```
int Foo(object o)
{
     return Bar(o);
}

object Bar(int i) => null;
```

In this case, we would be both updating outside the `Bar` call and *inside* it as well.

Reply:
From I know, when we invoke `FixAll` function, it only works for diagnostics with the same type of error. `return [||]Bar(o);` is error `CS0266`, and `return Bar([||]o)` is error `CS1503`. Since they are not the same error, they cannot be fixed at the same time using `FixAll`.

I have a test for this case (error 1503):
https://github.com/dotnet/roslyn/blob/38167d5fdbb8081ad0b3be1e3f88374f094cb3b6/src/EditorFeatures/CSharpTest/Diagnostics/AddExplicitCast/AddExplicitCastTests_FixAllTests.cs#L948

here is the result:
https://github.com/dotnet/roslyn/blob/38167d5fdbb8081ad0b3be1e3f88374f094cb3b6/src/EditorFeatures/CSharpTest/Diagnostics/AddExplicitCast/AddExplicitCastTests_FixAllTests.cs#L1075